src/modules/mavlink/streams/ATTITUDE_TARGET.hpp

```diff
@@ -100,7 +100,7 @@ class MavlinkStreamAttitudeTarget : public MavlinkStream
 			msg.body_pitch_rate = att_rates_sp.pitch;
 			msg.body_yaw_rate = att_rates_sp.yaw;
 
-			msg.thrust = att_sp.thrust_body[0];
+			msg.thrust = att_sp.thrust_body[2];
```

Comment:
```suggestion
			msg.thrust = Vector3f(att_sp.thrust_body).norm();
```

How about this? Then it works for both default multicopters and planes. It's also not clear what to really do for hybrid VTOLs, but I think this would work fine for now.

Reply:
I think this would work yes. Would it be worth doing something like [this](https://github.com/PX4/PX4-Autopilot/blob/e51000c5be901fc53bb2f5f63b82f9b9074316ea/src/modules/mavlink/mavlink_receiver.cpp#L1399) instead? This way the thrust setting and reporting matches for each vehicle type?

Reply:
Yes that would be fine, although vehicle types keep getting added without updating all these cases. The norm is the trivial one liner that should cover everything up until special cases that aren't really covered by this Mavlink message.

Reply:
Sounds good let's do that then 👍 Thanks for the suggestion.